backend/group_service/group/app/group_application_service.py

```diff
@@ -1,3 +1,17 @@
-# TODO: implement me :-)
+import logging
+
+from backend.group_service.group.domain.group import Group
+
+logger = logging.getLogger(__name__)
+
 class GroupApplicationService:
-    pass
+    def createGroup(self, riders, from_location, to_location):
+        return Group.objects.create(riders=riders, driver=None, 
+            from_location=from_location, to_location=to_location)
+
+    def updateGroup(self, groupId, driver):
+        myGroup = Group.objects.get(id=groupId)
+        myGroup.driver = driver
+        return myGroup
```

Comment:
Use update_group, group_id, driver_id
We need to find driver with driver_id => Driver.objects.get(id=driver_id)